<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>446</width>
    <height>319</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Вперёд</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ваш первый питомец:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_selected">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрано:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Кошка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Собака</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Черепаха</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Попугай</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Рыба</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хомяк</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Другое</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
